client/me/get-apps/index.jsx

```diff
@@ -10,83 +10,36 @@ import { localize } from 'i18n-calypso';
  */
 import MeSidebarNavigation from 'me/sidebar-navigation';
 import Main from 'components/main';
-import Button from 'components/button';
 import Card from 'components/card';
 import SectionHeader from 'components/section-header';
-
-export const GetApps = ( { translate } ) => (
-	<Main className="get-apps">
-		<MeSidebarNavigation />
-		<SectionHeader label={ translate( 'Desktop apps' ) } />
-		<Card className="get-apps__desktop">
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-osx.svg" alt={ translate( 'Mac' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Mac' ) }</strong><br />
-				{ translate( 'Requires 10.11 or newer' ) }</p>
-				<Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ translate( 'Download', { context: 'verb' } ) }</Button>
-			</section>
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-windows.svg" alt={ translate( 'Windows' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Windows' ) }</strong><br />
-				{ translate( 'Requires 7 or newer' ) }</p>
-				<Button href="https://apps.wordpress.com/d/windows?ref=getapps">{ translate( 'Download' ) }</Button>
-			</section>
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-linux.svg" alt={ translate( 'Linux' ) } width="96" height="96" />
-				</div>
-				<p><strong>{ translate( 'Linux' ) }</strong><br />
-				{ translate( 'Choose your distribution' ) }</p>
-				<Button href="https://apps.wordpress.com/d/linux?ref=getapps">
-					{ translate( '.TAR.GZ' ) }
-				</Button>
-				<Button href="https://apps.wordpress.com/d/linux-deb?ref=getapps">
-					{ translate( '.DEB' ) }
-				</Button>
-			</section>
-
-		</Card>
-
-		<SectionHeader label={ translate( 'Mobile apps' ) } />
-		<Card className="get-apps__mobile">
-
-			<section className="get-apps__app">
-				<div className="get-apps__app-icon">
-					<img src="/calypso/images/me/get-apps-ios.svg" alt={ translate( 'iOS' ) } width="96" height="96" />
+import GetAppsIllustration from './illustration.jsx';
+import DownloadCard from './download-card.jsx';
+
+export const GetApps = ( { translate } ) => {
+	const trackAppDownloadClick = () => {};
```

Comment:
Later when we want to track this let's use our [analytics](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics#examples) in redux over lib/analytics. The latter uses window without checking which is super annoying when testing. Ping me if you'd like some help connecting the component.